.travis.yml

```diff
@@ -24,9 +24,9 @@ install:
 before_script:
   - pip install -r dev-requirements.txt
   - python scripts/build_test_data.py
-  - PROFYLE_ingest --version
-  - PROFYLE_ingest tests/data/registry.db dataset1 tests/data/sample_clin_metadata.json
-  - PROFYLE_ingest tests/data/registry.db dataset1 tests/data/sample_pipe_metadata.json
+  - ingest --version
```

Comment:
This is the ingest command used in `candig-ingest`, for some reason, the version number of this `ingest` is `0.2`, which does not seem to be defined within the candig-ingest package..

Reply:
it's defined in the main.py of candig ingest using the docopts pkg